packages/web-component-library-angular/package.json

```diff
@@ -22,28 +22,33 @@
     "clean": "rimraf dist/"
   },
   "peerDependencies": {
-    "rxjs": "^6.6.0"
+    "@angular/common": "12 - 14",
+    "@angular/core": "12 - 14"
```

Comment:
Since Angular seems to be so great at backwards compatibility, I was thinking to use `>=11` instead of specifically limiting new versions beyond 14.